docs/source/el7toel8/envars.md

```diff
@@ -79,6 +80,39 @@ case they are changed. However, in some cases it's not wanted and it leads
 in malfunction network configuration (e.g. in case the bonding is configured
 on the system). It's expected that NICs have to be handled manually if needed.
 
+## LEAPP_DATABASE_FORCE_SYNC_ON
+
+If set to 1, leapp will explicitely synchronization on the SQLite database.
+Enabling the synchronisation has negative impact on the performance
+(sometimes very negative) however the data is more reliable in case of extreme
+situations (e.g. lost power).
+Note the synchronisation is nowadays switched off by default only during the phases
+executed before the reboot of the system to the upgrade environment which we consider
+safe, so we do not expect that anyone will want to use this option now.
+However, we are testing and discussing the possibility to switch off
+the syncrhonisation during the whole upgrade procedure in future
```

Comment:
```suggestion
If set to 1, leapp will explicitly enable synchronization on the SQLite database.
Enabling the synchronization has negative impact on the performance
(sometimes very negative) however the data is more reliable in case of extreme
situations (e.g. lost power).
Note the synchronization is nowadays switched off by default only during the phases
executed before the reboot of the system to the upgrade environment which we consider
safe, so we do not expect that anyone will want to use this option now.
However, we are testing and discussing the possibility to switch off
the syncrhonization during the whole upgrade procedure in future
```

Reply:
Adding fixup commit